packages/smooth_app/lib/helpers/analytics_helper.dart

```diff
@@ -114,7 +116,7 @@ class AnalyticsHelper {
       _crashReports = crashReports;
 
   static Future<void> setAnalyticsReports(final bool allow) async {
-    await MatomoTracker.instance.setOptOut(optout: !allow);
+    _allow = allow;
```

Comment:
```suggestion
    _allow = allow;
    await MatomoTracker.instance.setOptOut(optout: false);
```
Now that I'm thinking, I'm not sure about the default value of matomo's `optout`.
At least with that extra command it's explicit.